lexicon.md

```diff
@@ -0,0 +1,110 @@
+
+### Exploitable Vulnerabilities
+
+[Defects](#defects) in the software that can be leveraged by attackers to gain unauthorized access, execute [arbitrary code](#arbitrary-code), or cause other undesired outcomes.
+
+### License
+
+A legal document that defines the terms under which the software can be used, modified, and distributed. May be stored at the top level of the [repository](#repository) in a file named `LICENSE` or within files in a directory named `LICENSE/`. The license applies to the [codebase](#codebase) and any [released software assets](#released-software-assets), unless otherwise stated.
+
+### Known Vulnerabilities
+
+Publicly acknowledged [exploitable vulnerabilities](#exploitable-vulnerabilities) that have been identified within the software. These vulnerabilities often have associated advisories, patches, or recommended mitigations.
+
+### Multi-factor Authentication (MFA)
+
+An authentication method that requires two or more verification factors (e.g., a password and a token) to gain access to a resource. This method strengthens security by requiring multiple forms of identification.
+
+### Primary Branch
+
+The main development branch in the [version control system](#version-control-system), representing the latest stable codebase. [Releases](#release) are typically made from this branch. Commonly named `main` or `master`. In some situations where branches are not featured, a [repository](#repository) with forked repositories will have the original repo acting as as an equivalent to the primary branch.
+
+### Project Documentation
+
+Written materials related to the project, such as user guides, developer guides, and contribution guidelines. These documents help users and developers understand, contribute to, and interact with the software. At release time, this may include [provenance] information, licensing details, and other metadata.
+
+### Provenance
+
+Information about the origin and history of the [released software assets](#released-software-assets). This may include details about its development, dependencies, vulnerabilities, contributors, and licensing.
+
+### Release
+
+- _(verb)_ The process of making a version-controlled bundle of assets available to users.
+- _(noun)_ A version-controlled bundle of code, [documentation](#project-documentation), and other assets made available to users. A release often includes release notes that describe the changes.
+
+### Released Software Assets
+
+Deliverables provided to users as part of a [release](#release). These assets can include binaries, libraries, or containers.
+
+### Releases
+
+Specific versions of the software that have been made available to users, typically accompanied by a [unique version identifier](#unique-version-identifier). Each release includes notes on updates and changes.
```

Comment:
```suggestion
Specific versions of the software that have been made available to users, typically accompanied by a [unique version identifier](#unique-version-identifier). Each [release](#release) includes notes on updates and changes.
```

Reply:
I would link this back to the definition of "release" that is just few headings above. TBH I am not sure it is necessary to make different definitions for single and plural, as they sound a bit redundant to me. What was the rationale behind this?

Reply:
This was an oversight :D Thanks for catching